experiments/list_unsupported.sh

```diff
@@ -88,7 +87,21 @@ else
       exit 9
    fi
 fi
+
+# Can we build the support tool?
+# Problem: GNAT 2016 helpfully installs a g++ binary alongside gnat... except
+# it's way out of date so invoking g++ via PATH looking will pickup that gnat g++
```

Comment:
FWIW we should probably also update our gnat to a newer release at some point

Reply:
Yes although as I understand it this is a non-trivial job because there is a dependency between the version of GNAT that gnat2goto is based on and the version we use and 'rebasing' gnat2goto may well be non-trivial.  I am inclined to say it is not a task for this project.